upstream-community-operators/keycloak-operator/11.0.0/keycloak-operator.v11.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,356 @@
+              - routes
+            verbs:
+              - list
+              - get
+              - create
+              - update
+              - watch
+          - apiGroups:
+              - extensions
+            resources:
+              - ingresses
+            verbs:
+              - list
+              - get
+              - create
+              - update
+              - watch
+          - apiGroups:
+              - monitoring.coreos.com
+            resources:
+              - servicemonitors
+              - prometheusrules
+              - podmonitors
+            verbs:
+              - list
+              - get
+              - create
+              - update
+              - watch
+          - apiGroups:
+              - integreatly.org
+            resources:
+              - grafanadashboards
+            verbs:
+              - get
+              - list
+              - create
+              - update
+              - watch
+          - apiGroups:
+              - apps
+            resourceNames:
+              - keycloak-operator
+            resources:
+              - deployments/finalizers
+            verbs:
+              - update
+          - apiGroups:
+              - policy
+            resources:
+              - poddisruptionbudgets
+            verbs:
+              - get
+              - list
+              - create
+              - update
+              - watch
+          - apiGroups:
+              - keycloak.org
+            resources:
+              - keycloaks
+              - keycloaks/status
+              - keycloaks/finalizers
+              - keycloakrealms
+              - keycloakrealms/status
+              - keycloakrealms/finalizers
+              - keycloakclients
+              - keycloakclients/status
+              - keycloakclients/finalizers
+              - keycloakbackups
+              - keycloakbackups/status
+              - keycloakbackups/finalizers
+              - keycloakusers
+              - keycloakusers/status
+              - keycloakusers/finalizers
+            verbs:
+              - get
+              - list
+              - update
+              - watch
+          serviceAccountName: keycloak-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: false
+      type: AllNamespaces
+  keywords:
+    - Keycloak
+    - Identity
+    - Access
+  links:
+    - name: Documentation
+      url: https://github.com/keycloak/keycloak-operator/tree/11.0.0/docs
+    - name: Keycloak
+      url: https://www.keycloak.org/
+    - name: Keycloak Discourse
+      url: https://keycloak.discourse.group/
+  maintainers:
+    - email: keycloak-dev@lists.jboss.org
+      name: Keycloak DEV mailing list
+  maturity: alpha
+  provider:
+    name: Red Hat
+  version: 11.0.0
+  replaces: keycloak-operator.v10.0.2
```

Comment:
There is no `keycloak-operator.v10.0.2`. There is `keycloak-operator.v10.0.0` to replace. Can you fix it?

Reply:
@mvalahtv Thank you, done.